app/presenters/hyku/menu_presenter.rb

```diff
@@ -23,7 +23,7 @@ def roles_and_permissions_section?
     # Returns true if the current controller happens to be one of the controllers that deals
     # with repository activity  This is used to keep the parent section on the sidebar open.
     def repository_activity_section?
-      %w(admin status).include?(controller_name)
+      %w(admin dashboard status).include?(controller_name)
```

Comment:
Please add a test for this.  Mostly I want the test to make sure that if `Hyrax::DashboardController` ever goes away, we can remember to clean this up.